pipeline-scripts/workload.groovy

```diff
@@ -21,16 +23,55 @@ stage (workload) {
 	// get properties file
 	sh "wget ${workload_properties_file} -O ${property_file_name}"
 	sh "cat ${property_file_name}"
-
 	// Load the properties file.
 	def properties = readProperties file: property_file_name
 	def job_parameters = []
 	job_parameters.add([$class: 'LabelParameterValue', name: 'node', label: node_label ])
-	
 	// Convert properties to parameters.
 	for (property in properties) {
 		job_parameters.add([$class: 'StringParameterValue', name: property.key, value: property.value ])
 	}
+        
+	if (pipeline == "TRUE") {
+		if (workload == "ATS-SCALE-CI-TEST") {
```

Comment:
s/ATS-SCALE-CI-TEST/ATS-SCALE-CI-SCALE

Reply:
In a follow on lets just S/ATS-//g

Reply:
yup, will do a bit of a cleanup in a different PR